docker-compose.yml

```diff
@@ -5,7 +5,7 @@ redis:
 db:
   image: orchardup/postgresql
   ports:
-    - 5432:5432
+    - 4040:5432
```

Comment:
Please remove this change from the PR so we don't change the port for everyone else who already has things set up